packages/caisar/caisar.0.1/opam

```diff
@@ -0,0 +1,62 @@
+opam-version: "2.0"
+synopsis:
+  "A platform for characterizing the safety and robustness of artificial intelligence based software"
+maintainer: [
+  "LAISER team, Software Safety and Security Laboratory, CEA-List"
+]
+authors: ["LAISER team, Software Safety and Security Laboratory, CEA-List"]
+license: "LGPL-2.1-only"
+homepage: "https://git.frama-c.com/pub/caisar"
+doc: "https://git.frama-c.com/pub/caisar"
+bug-reports: "https://git.frama-c.com/pub/caisar/issues"
+depends: [
+  "ocaml" {>= "4.13"}
+  "dune-site" {= "2.9.0"}
+  "piqi" {>= "0.7.6"}
+  "piqilib" {>= "0.6.14"}
+  "zarith" {>= "1.7"}
+  "ocplib-endian" {>= "1.0"}
+  "dune" {>= "2.9" & >= "2.9.3"}
```

Comment:
The first one is unnecessary
```suggestion
  "dune" {>= "2.9.3"}
```

Reply:
I agree it seems unnecessary, but this is something generated by `dune`. Suggestions?

Reply:
What I meant is that, whenever I would rebuild with `dune`, `dune` will re-generate the `opam` files with `{>= "2.9" & >= "2.9.3"}` (I ran some test), so changing the `opam` file by hand seems not a good solution to me.

Reply:
you just need to remove the dune dependencies from the `dune-project` file.
The other possibility is to stop generating the opam file by dune, if you were already ready to update the constraints needed by hand there is nothing more dune does add to that.